lightwood/mixers/nn.py

```diff
@@ -714,6 +719,11 @@ def calculate_accuracy(self, ds):
                 reals = [str(x) for x in ds.get_column_original_data(output_column)]
                 preds = [str(x) for x in predictions[output_column]['predictions']]
 
+            if isinstance(self.net, ArNet):
+                # for time series, we only consider loss of rows w/full historical context
+                reals = reals[len(self.net.ar_idxs):]
```

Comment:
will ar_idxs always be < len(reals) or len(pred)?

Reply:
It should, I'll confirm and report back.